nature/migrations/0004_field_updates.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.4 on 2017-10-31 09:25
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import nature.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('nature', '0003_update_timestamp_fields'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='event',
+            name='protection_level',
+            field=models.IntegerField(choices=[(1, 'Administrators only'), (2, 'Administrators and staff'), (3, 'Public')], db_column='suojaustasoid', default=3),
+        ),
+        migrations.AlterField(
+            model_name='feature',
+            name='active',
+            field=models.BooleanField(db_column='voimassa', default=True),
+        ),
+        migrations.AlterField(
+            model_name='feature',
+            name='geometry1',
+            field=nature.models.PermissiveGeometryField(db_column='geometry1', srid=3879),
```

Comment:
Here, we have a problem that the migration won't go through on the production database, because we have postgres views defined on the `geometry1` column. That is because we also have geoserver fetching data on the database, and our geoserver uses postgres views to fetch the layers it distributes in WFS. Apparently, columns with views on them cannot be altered: 

```
psycopg2.NotSupportedError: cannot alter type of a column used by a view or rule
DETAIL:  rule _RETURN on view ltj_avoin.vesi_vesikasvilinjat depends on column "geometry1"
```

Any suggestions on how we could add the default srid on the field, do we have to drop the views (which results in api downtime) and create them again manually?

Reply:
Ah, I see. We are not changing the schema actually, what we need is just rename the field. We can put the right field definition in the initial migration for the geometry field and reapply the migrations again.